### ISSUE TITLE
Add accessibility guidelines for frontend developers

### ISSUE THREAD

**Issue by felixheidecke** [owncloud/docs#296]
Add general guidelines on how to ensure that all apps are conforming to [WCAG](https://www.w3.org/standards/techs/wcag#w3c_all) and employ WAI-ARIA in the DOM.

No need for many words but a couple of links to must-read documents as well as a few examples.

**Note:** this should become a part of App/Code-Reviews

**Comment by settermjd** [comment]
@felixheidecke - any time to progress this further?

**Comment by settermjd** [comment]
ping @felixheidecke

**Comment by felixheidecke** [comment]
Hi there ... there isn't any progress here. And since I'm currently working on legacy code ... I doubt I'll come back for it ⊙﹏⊙∥
